### PR TITLE
Milestone 5: Event Handlers

### PR DESCRIPTION
**Pull request by taiwocoker**
In this milestone:
- I implemented click handler in ButtonPanel and Button component
- I implemented the render function in the App component
- I implemented props drilling passing the handle click function from App.js to button.js

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import React, { Component } from 'react';
 import '../index.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// eslint-disable-next-line no-unused-vars
```
Comment (Torres-ssf) [inline src/components/App.js:5]: - [ ] We can safely remove this eslint disable line since you do not have any unused vars here.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @taiwocoker :hand: 
Awesome work on this milestone so far :clap: :+1: 
We only need to make some changes before moving on :smile: 

- [ ] We need to deal with division by 0. Right now the app throws an error.

- [ ] I notice a bug in your application, it seems sometimes the calculator does not work as expected and the result button does not show the result of the operation. I must emphasize however that this is not always the case, sometimes works as expected, sometimes don't
bug1

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @taiwocoker 👋

**You've done quite well on this.👍**


**STATUS: CHANGES REQUIRED. 👷‍♀️**

- [ ] However, consider the fact that it would improve user experience if the result from a previous calculation could be used as the first input for the next calculation. Kindly add that feature.
chrome-capture

- [ ] In addition, as requested in the previous review kindly fix [this issue](https://github.com/taiwocoker/React-Calculator/pull/5#discussion_r499608012).

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by shloch — APPROVED** [review]
## Hello @taiwocoker , 

Great job......points from former TSEs have been taken into consideration
Keep up with the good work
calc

It's thumbs up for me

**Pull request merged**